src/RuleSet/Sets/PhpCsFixerSet.php

```diff
@@ -119,7 +119,7 @@ public function getRules(): array
             'single_line_comment_style' => true,
             'single_line_empty_body' => true,
             'single_line_throw' => false,
-            'string_implicit_backslashes' => ['single_quoted' => 'ignore'],
```

Comment:
I see change of config as something unexpected, I would need to ask wider question in first place. That is not directly scope of this PR, but needed for me to align how we want to move on here.

why `string_implicit_backslashes` is having so many options? why for each syntax we allow to escape/ignore/unescape?

Why not just always escape for double_quoted and heredoc, always unescape for single_quoted (and maybe, maybe, allow to set them to `ignore`) ?

Out of the blue, I would rather deprecate all options of the rule, looking at it.

Reply:
Configs options are for users that want different syntax or possibly needs to align the syntax with another CS fixing tools.

In this PR, let's focus solely in normalizing single quotes. All other string notations are fixed already.

Reply:
I want to normalize only for direction that we consider the best practice.
do we have the best practice aligned? is it the current defaults of those options?

Reply:
Yes - this PR is to unquote possibly quoted single string backslashes and this is strongly the used style in this project and IMO other projects too.